src/style.css

```diff
@@ -1,3 +1,88 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./font/Roboto-Medium.ttf);
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: gray;
+  display: flex;
+  justify-items: center;
+  font-size: 12px;
+}
+
+header {
+  width: 100%;
+  background-color: rgba(255, 255, 255, 1);
+  text-align: center;
```

Comment:
Will something change if you remove this rule? 
Looks like a redundant css.

Reply:
Brak css dla headera niszczy leayout.